.github/actions/setup-ubuntu-host/action.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Setup Ubuntu Runner Host'
+inputs:
+  python-version:
+    description: 'Python version to use'
+    required: true
+    default: '3.13'
+runs:
+  using: "composite"
+  steps:
+    - name: setup python
+      uses: actions/setup-python@v5
+      with:
+        python-version: ${{inputs.python-version}}
+        cache: 'pip' # caching pip dependencies
+    - name: Install dependencies
+      shell: bash
+      run: |
+        sudo apt-get install libldap2-dev libsasl2-dev # Required for python-ldap
```

Comment:
Q: Why do we need python-ldap for release notes?

Reply:
We require installed python to run the release notes script. Python dependencies are defined in `requirements.txt`. One of the dependencies is `python-ldap`.

Reply:
So `python-ldap` is not required by release notes tool itself, but by something else (perhaps something we use in tests) and we only need it because we combined dependencies of the release note tool with other python dependencies. Do I understand correctly?

Reply:
yes, we have a single `requirement.txt` file for all python scripts, including e2e tests, pipeline.py and all scripts in `scripts/` dir

Reply:
Got it. It might be worth separating release note tool requirements from others, but I don' think it is critical to do now.

Resolving this question.